mayflower/src/main/java/module-info.java

```diff
@@ -17,6 +18,8 @@
     exports io.github.madethoughts.mayflower.plugin;
     exports io.github.madethoughts.mayflower.listener;
     exports io.github.madethoughts.mayflower.lifecycle.event;
+    exports io.github.madethoughts.mayflower.configuration;
+    exports io.github.madethoughts.mayflower.lifecycle.event.internal;
```

Comment:
Are you sure we want to export it? Or do you say that the package indicates that it is internal and is suspect to breaking changes without major version updates?